ns1/resource_record.go

```diff
@@ -473,6 +504,28 @@ func metaDiffSuppress(k, old, new string, d *schema.ResourceData) bool {
 		return len(compareMap) == 0
 	}
 
+	if metaDiffSuppressUp(k, old, new, d) {
+		return true
+	}
+
+	return false
+}
+
+func metaDiffSuppressUp(k, old, new string, d *schema.ResourceData) bool {
```

Comment:
Is `d` unused in this func?

Reply:
yeah, gotta have it in the function signature since it's used as a DiffSuppressFunc though. I'll swap it to underscore though.